src/tests/test25-dynamic-backend-address.sh

```diff
@@ -15,13 +15,25 @@ cat >hitch.cfg <<EOF
 pem-file = "${CERTSDIR}/default.example.com"
 frontend = "[localhost]:$LISTENPORT"
 backend = "[hitch_test]:80"
+backend-refresh = 1
 EOF
 
 start_hitch --config=hitch.cfg
-curl_hitch
 
-echo "hitch_test varnish-cache.org" >hosts
+# TODO curl_hitch - cannot grep in output
```

Comment:
@dirid: can you help me with porting my test to use your functions? I couldn't figure out how to parse the output of `run_curl`. I'd like to achieve the equivalent of this.

```
curl --max-time 5  --insecure https://localhost:$LISTENPORT/ >backend_refresh2.dump 2>&1                      
run_cmd grep -q "We moved the Varnish Project to a new server" backend_refresh2.dump   
```

Reply:
See https://github.com/varnish/hitch/commit/c9c2069fcf3af6d3dfc25ec57a73e4f4579c6e33#diff-eb92142e01d0e47c57b2ed4c18d6e2d3L30

Reply:
404 is not the case: before & after the backend address change, curl will succeed.
The difference is the returned page, before it's hitch-tls.org then varnish-cache.org.

The test needs to parse(grep in) the returned page.